src/diffusers/schedulers/scheduling_karras_ve.py

```diff
@@ -103,6 +103,11 @@ def step(
         sigma_prev: float,
         sample_hat: Union[torch.FloatTensor, np.ndarray],
     ):
+        if self.num_inference_steps is None:
```

Comment:
same here I don't think we should throw an error if we don't have to - I'd be in favor of removing it here